filters/ignore_new_executors.cpp

```diff
@@ -0,0 +1,60 @@
+#include "glog/logging.h"
+
+#include "filters/ignore_new_executors.hpp"
+
+#include "stout/try.hpp"
+#include "stout/nothing.hpp"
+
+namespace mesos {
+namespace serenity {
+
+Try<Nothing> IgnoreNewExecutorsFilter::consume(const ResourceUsage &usage) {
+  std::unique_ptr<ExecutorMap<time_t>> newExecutorTimestamps =
+    std::unique_ptr<ExecutorMap<time_t>>(new ExecutorMap<time_t>());
+  ResourceUsage newUsage;
```

Comment:
Not mutable resourceUsage, but rather copy unique_ptrs instead of whole structure.